Blazor_Foundation_6_Test_Shared/Documentation/Containers/Off-Canvas.razor

```diff
@@ -1,6 +1,416 @@
+
+        <GridX>
+            <Cell Size="small-6">
+                <OpenCodeDev.Blazor.Foundation.Components.Containers.OffCanvasWrapper>
+                    <OpenCodeDev.Blazor.Foundation.Components.Containers.OffCanvas Absolute="true" Position="position-left" Id="offCanvasLeftSplit1">
+                        <ul class="vertical menu">
+                            <li><a href="#">Home</a></li>
+                            <li><a href="#">Service</a></li>
+                            <li><a href="#">About</a></li>
+                            <li><a href="#">Contact</a></li>
+                        </ul>
+                    </OpenCodeDev.Blazor.Foundation.Components.Containers.OffCanvas>
+
+
+                    <OpenCodeDev.Blazor.Foundation.Components.Containers.OffCanvasContent style="min-height: 300px;">
+                        <p>I have nothing to do with the off-canvas on the right!</p>
+                    </OpenCodeDev.Blazor.Foundation.Components.Containers.OffCanvasContent>
+                </OpenCodeDev.Blazor.Foundation.Components.Containers.OffCanvasWrapper>
+            </Cell>
+            <Cell Size="small-6">
+                <OpenCodeDev.Blazor.Foundation.Components.Containers.OffCanvasWrapper>
+                    <OpenCodeDev.Blazor.Foundation.Components.Containers.OffCanvas Absolute="true" Position="position-right" Id="offCanvasRightSplit2" style="background-color: #000000">
+                        <ul class="vertical menu" style="color: #ffffff">
+                            <li><a href="#">Home</a></li>
+                            <li><a href="#">Service</a></li>
+                            <li><a href="#">About</a></li>
+                            <li><a href="#">Contact</a></li>
+                        </ul>
+                    </OpenCodeDev.Blazor.Foundation.Components.Containers.OffCanvas>
+                    <OpenCodeDev.Blazor.Foundation.Components.Containers.OffCanvasContent style="min-height: 300px;">
+                        <p>Im a unique off-canvas container all on my own!</p>
+                    </OpenCodeDev.Blazor.Foundation.Components.Containers.OffCanvasContent>
+                </OpenCodeDev.Blazor.Foundation.Components.Containers.OffCanvasWrapper>
+            </Cell>
+        </GridX>
+        <hr />
+        <h2>Off-canvas Directions</h2>
+        <p>
+            Off-canvas can open from any direction, left, right, top, and bottom.
+        </p>
+        <p>
+            The Off-canvas component requires a positioning attribute to determine which side of the viewport it opens from:
+            <ul>
+                <li><Code>position-left</Code></li>
+                <li><Code>position-right</Code></li>
+                <li><Code>position-top</Code></li>
+                <li><Code>position-bottom</Code></li>
+            </ul>
+        </p>
+        <HighlightCS Language="html" Full="true" HeaderIcon="mdi-language-html5" HeaderBGcolor="#5d2b90" HeaderTitle="Razor"
```

Comment:
The code example has an img tag artifact either add an image or code that part of the code.